projects/core/src/occ/occ-models/occ.models.ts

```diff
@@ -4337,4 +4343,74 @@ export namespace Occ {
     pagination?: PaginationModel;
     sorts?: SortModel[];
   }
+
+  export interface ReplenishmentOrder {
+    active?: boolean;
+    appliedOrderPromotions?: PromotionResult[];
+    appliedProductPromotions?: PromotionResult[];
+    appliedVouchers?: Voucher[];
+    calculated?: boolean;
+    code?: string;
+    costCenter?: CostCenter;
+    deliveryAddress?: Address;
+    deliveryCost?: Price;
+    deliveryItemsQuantity?: number;
+    deliveryMode?: DeliveryMode;
+    deliveryOrderGroups?: DeliveryOrderEntryGroup[];
+    description?: string;
+    entries?: OrderEntry[];
+    expirationTime?: Date;
+    firstDate?: Date;
+    guid?: string;
+    name?: string;
+    net?: boolean;
+    orderDiscounts?: Price;
+    paymentInfo?: PaymentDetails;
+    paymentStatus?: string;
+    paymentType?: B2BPaymentType;
+    pickupItemsQuantity?: number;
+    pickupOrderGroups?: PickupOrderEntryGroup[];
+    potentialOrderPromotions?: PromotionResult[];
+    potentialProductPromotions?: PromotionResult[];
+    productDiscounts?: Price;
+    purchaseOrderNumber?: string;
+    replenishmentOrderCode?: string;
+    saveTime?: Date;
+    savedBy?: Principal;
+    site?: string;
+    store?: string;
+    subTotal?: Price;
+    totalDiscounts?: Price;
+    totalItems?: number;
+    totalPrice?: Price;
+    totalPriceWithTax?: Price;
+    totalTax?: Price;
+    totalUnitCount?: number;
+    user?: Principal;
+  }
+
+  export interface ReplenishmentOrderList {
+    pagination?: PaginationModel;
+    replenishmentOrders?: ReplenishmentOrder[];
+    sorts?: SortModel[];
+  }
+
+  export interface ScheduleReplenishmentForm {
+    daysOfWeek?: DaysOfWeek[];
+    nthDayOfMonth?: string;
+    numberOfDays?: string;
+    numberOfWeeks?: string;
+    recurrencePeriod?: string;
+    replenishmentStartDate?: string;
+  }
+
+  export enum DaysOfWeek {
+    SUNDAY = 'SUNDAY',
```

Comment:
Just curious, you might not know how does this work with translations? I assume the API is always English and you would do a mapping for the date picker dialog thing?

Reply:
Good question about the language :), i did not think of that, but yes a mapping will suffice in the date picker dialog thing.

But, yes I believe the API only accepts english dates as I checked in the `b2b-accelerator` that it is those specific enums